kubernetes/tests/kubernetes/kubernetes.tests.bom

```diff
@@ -23,35 +23,81 @@ brooklyn.catalog:
           brooklyn.config:
             endpointSensor: main.uri
             timeout: 5m
+        - type: org.apache.brooklyn.test.framework.TestSensor
+          name: "3. Size of kubernetes master cluster is 3"
```

Comment:
This may be the most nit-picky review point ever made but now the numbering goes 01 02 3 4.  I won't take it personally if you ignore this.